eng/scripts/Language-Settings.ps1

```diff
@@ -204,24 +204,25 @@ function DockerValidation{
     [string]$workingDirectory
   ) 
   if ($PackageSourceOverride) {
-    docker run -v "${workingDirectory}:/workdir/out" -e TARGET_PACKAGE=$packageName -e TARGET_VERSION=$packageVersion `
+    $commandLine = docker run -v "${workingDirectory}:/workdir/out" -e TARGET_PACKAGE=$packageName -e TARGET_VERSION=$packageVersion `
        -e EXTRA_INDEX_URL=$PackageSourceOverride -t $DocValidationImageId 2>&1 | Out-Null
   }
   else {
-    docker run -v "${workingDirectory}:/workdir/out" `
+    $commandLine = docker run -v "${workingDirectory}:/workdir/out" `
       -e TARGET_PACKAGE=$packageName -e TARGET_VERSION=$packageVersion -t $DocValidationImageId 2>&1 | Out-Null
   }
   # The docker exit codes: https://docs.docker.com/engine/reference/run/#exit-status
   # If the docker failed because of docker itself instead of the application, 
   # we should skip the validation and keep the packages. 
+  
   if ($LASTEXITCODE -eq 125 -Or $LASTEXITCODE -eq 126 -Or $LASTEXITCODE -eq 127) { 
     Write-Host $commandLine
```

Comment:
I think this is always going to be empty because you are piping to `Out-Null`. You should remove the null pipe if you're capturing it in a variable to be printed.

Reply:
I will put the message to debug level. Then, we can rerun the pipeline by turn on 'System.debug'